go.mod

```diff
@@ -36,6 +35,7 @@ require (
 	github.com/lucasb-eyer/go-colorful v1.2.0 // indirect
 	github.com/magiconair/properties v1.8.7 // indirect
 	github.com/mailru/easyjson v0.7.7 // indirect
+	github.com/massdriver-cloud/terraform-config-inspect v0.0.1 // indirect
```

Comment:
Strange this is in the indirect block. This should be in the top immediate dependency block. Did you run `go get github.com/massdriver-cloud/terraform-config-inspect`?

Reply:
I did. After using `go mod tidy`, it seems to have moved over